common/characters/AFG.txt

```diff
@@ -765,22 +731,26 @@ characters = {
 			allowed = {
 				original_tag = AFG
 			}
-			visible = {
-				NOT = {
-					has_global_flag = afg_dlc_focus_tree
-				}
-			}
 			available = {
 				OR = {
 					has_government = fascism
-					AND = {
+					if = {
+						limit = { has_global_flag = afg_dlc_focus_tree }
 						has_country_leader = {
 							character = AFG_amanullah_khan
 							ruling_only = yes
 						}
-						has_country_flag = AFG_approached_haji_mirzali_khan
-						NOT = {
-							has_country_flag = AFG_dissatisfied_haji_mirzali_khan
+					}
+					else = {
+						AND = {
```

Comment:
Doesn't have be adjust but be aware that inside `if` statements, AND is the default again, even if the scope outside it was `OR`